### PR TITLE
feat(azure): small improvements for PR assignees and reviewers (corre…

### PR DESCRIPTION
**Pull request by kroonprins**
…ctly assign when display name format is used; also add assignees and reviewers for onboarding PR if assigness/reviewers have been provided as command line arguments)

For Azure PR assignees are added by doing a mention in a PR comment. Currently when the assignee is given by its "displayName" instead of its "uniqueName" the mention will not work correctly. For reviewers it is possible to configure the reviewers by both their "displayName" and "uniqueName", so the code change of this pull request applies the same logic for reviewers also to assignees. Being able to specify assignees by display name can be useful because Azure Devops pipelines provide the display name as an environment variable in a running pipeline, but not the unique name.

### CODE REVIEW

File: lib/platform/azure/index.ts [hunk 1]
```diff
@@ -639,33 +639,7 @@ export /* istanbul ignore next */ function getIssueList(): Promise<Issue[]> {
...
- *
- * @param {number} prNo
- * @param {string[]} reviewers
- */
-export async function addReviewers(
-  prNo: number,
-  reviewers: string[]
-): Promise<void> {
-  logger.trace(`addReviewers(${prNo}, ${reviewers})`);
+async function getUserIds(users: string[]): Promise<any[]> {
```
Comment (viceice) [inline lib/platform/azure/index.ts]: @JamieMagee Can you add explicit types please

**Review by viceice — APPROVED** [review]
LGTM

File: lib/workers/repository/onboarding/pr/index.ts [hunk 2]
```diff
@@ -128,6 +129,7 @@ If you need any further assistance then you can also [request help here](${confi
         useDefaultBranch,
       });
       logger.info({ pr: pr.displayNumber }, 'Onboarding PR created');
+      await addAssigneesReviewers(config, pr);
```
Comment (viceice) [inline lib/workers/repository/onboarding/pr/index.ts]: @rarkins I think this should be done in separate pr.

**Review by viceice — CHANGES_REQUESTED** [review]
Forgot about the second change

File: lib/workers/repository/onboarding/pr/index.ts (hunk 2 above)
Comment (rarkins) [reply lib/workers/repository/onboarding/pr/index.ts]: Yes, please separate this to another PR

Comment (kroonprins) [reply lib/workers/repository/onboarding/pr/index.ts]: Updates related to second change removed with [this commit](https://github.com/renovatebot/renovate/pull/5601/commits/032e6dbbe4580f02c6f3fad78ee148ef136d9e03). New PR opened for it [here](https://github.com/renovatebot/renovate/pull/5611).

**Review by JamieMagee — APPROVED** [review]

**Comment by JamieMagee** [comment]
@viceice Needs your approval

**Review by viceice — APPROVED** [review]
